### ISSUE TITLE
File in the current directory is not found without a leading ./

### ISSUE THREAD

**Issue by Sworddragon** [davidben/nspluginwrapper#50]
I'm using nspluginwrapper 1.4.4 and it seems nspluginwrapper can't find relative files without a leading ./. Here is what happens:

> sworddragon@ubuntu:~/.mozilla/plugins$ nspluginwrapper -v -u npwrapper.libflashplayer.so
> nspluginwrapper: npwrapper.libflashplayer.so is not a valid nspluginwrapper plugin

But using a leading ./ results in this:

> sworddragon@ubuntu:~/.mozilla/plugins$ nspluginwrapper -v -u ./npwrapper.libflashplayer.so
> Update plugin ./npwrapper.libflashplayer.so
>   NPAPI plugin /home/sworddragon/.mozilla/plugins/libflashplayer.so was modified, reinstalling plugin
> Install plugin /home/sworddragon/.mozilla/plugins/libflashplayer.so
>   into /home/sworddragon/.mozilla/plugins/npwrapper.libflashplayer.so

**Comment by davidben** [comment]
So, I haven't touched this project for years now. I'm not sure anyone else has either. The landscape is pretty different now with Chrome dropping NPAPI altogether altogether, 64-bit Flash existing, and out-of-process plugins being standard fare in browsers which still support them.

What are you using nspluginwrapper for?

**Comment by Sworddragon** [comment]
> The landscape is pretty different now with Chrome dropping NPAPI altogether

Sure that Chrome plans to drop NPAPI support? As I know they want to make it just deprecated but not drop it.

> What are you using nspluginwrapper for?

I need it for the debug version of the Flash Player. For Firefox on Linux only the Flash Player 11.2.x (NPAPI) is available and as I know the debug version of the Flash Player is only available from Adobe as 32 bit.

**Comment by davidben** [comment]
Yes, NPAPI is gone in Chrome now. And it's been gone from Linux for a while now. (Edit: To clarify, it's been gone from Chrome on Linux for even longer than Chrome for other platforms.) Flash in Chrome doesn't use NPAPI.

Interesting. I hadn't heard of this debug Flash. Yeah, I dunno. As far as I know, no one's working on this project anymore.
